### PR TITLE
Update issue chooser with forms and Discussions links

### PR DESCRIPTION
**Pull request by mdcfe**
This PR moves support tickets and feature requests from the issue tracker to Discussions, which provides a few benefits:
- The issue tracker can be used to track actual bugs/problems more cleanly, without being cluttered up with support tickets
- The Q&A category on Discussions lets users vote on comments and marking replies as answers, which makes it easier to find solutions to the same problem later on.
- Feature requests can be discussed/brainstormed to decide what features people want, and then can be superseded by an issue/PR once the feature request has been nailed down.

Possible downsides:
- Discussions don't support templates yet, but this feature is [now on GitHub's roadmap](https://github.com/github/feedback/discussions/2838).
  - We can set up a bot to auto-respond to new Q&A issues to welcome users and remind them to include important information.
- Existing issues may need converting to discussions.
  - This is temporarily disabled, but should be returning soon.
  - Once this is enabled again, it should be fairly trivial to skim over current open issues and convert to discussions.

### CODE REVIEW

**Comment by JRoy** [comment]
Probably should wait for templates to be added as a feature to discord since we cannot move users to categories via a link ourselves yet, this could cause possible miss-categorization. 

If we don't care about that though (think it won't be that big of a problem), I have no other objects.

**Review by JRoy — DISMISSED** [review]
caffeine deprivation

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+    about: Create a support ticket to get help from developers.
+  - name: Suggest a feature
+    url: https://github.com/EssentialsX/Essentials/discussions/new?category=ideas-and-feature-suggestions
+    about: Suggest new features for EssentialsX!
+  - name: View previous help requests
+    url: https://github.com/EssentialsX/Essentials/discussions/categories/q-a
+    about: Visit the help forum to search for existing help requests
+  - name: Discuss existing feature suggestions
+    url: https://github.com/EssentialsX/Essentials/discussions/categories/ideas-and-feature-suggestions
+    about: Visit the help forum to search for existing help requests
```
Comment (triagonal) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Visit the Feature Suggestions forum to discuss existing feature suggestions.
```
i'm bad at words - the actual point is the accidental unchanged copy-paste

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
-  - name: MOSS Discord Server
+  - name: Create help request
+    url: https://github.com/EssentialsX/Essentials/discussions/new?category=q-a
+    about: Create a support ticket to get help from developers.
+  - name: Suggest a feature
+    url: https://github.com/EssentialsX/Essentials/discussions/new?category=ideas-and-feature-suggestions
+    about: Suggest new features for EssentialsX!
+  - name: View previous help requests
+    url: https://github.com/EssentialsX/Essentials/discussions/categories/q-a
+    about: Visit the help forum to search for existing help requests
```
Comment (triagonal) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Visit the help forum to search for existing help requests.
```
missing punctuation

**Review by JRoy — DISMISSED** [review]

**Review by JRoy — DISMISSED** [review]

**Review by JRoy — APPROVED** [review]

**Pull request merged**